pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools-scm", "wheel==0.36.2", "auditwheel==4.0.0"]
+requires = ["setuptools-scm==6.0.1", "wheel==0.37.0", "auditwheel==4.0.0"]

```

Comment:
please exclude that from the PR

Reply:
@mxschmitt I had already created a pr to bump versions of which I will rebase so merge that first

Reply:
Please still revert it. Normally you always base-off from master.